packages/ui-react/src/components/Icon/Icon.tsx

```diff
@@ -0,0 +1,15 @@
+import classNames from 'classnames';
+import React, { FC, SVGProps } from 'react';
+
+import styles from './Icon.module.scss';
+
+export type Props = {
+  icon: FC<SVGProps<SVGSVGElement>>;
+  className?: string;
+};
+
+const Icon: FC<Props> = ({ icon: IconComponent, className }) => {
+  return <IconComponent className={classNames(styles.icon, className)} />;
```

Comment:
We need the `aria-hidden="true"` back 😄 I think we do not need the focusable="false" as reading some articles on internet. `aria-hidden="true"` is sufficient

```suggestion
  return <IconComponent aria-hidden="true" className={classNames(styles.icon, className)} />;
```

Reply:
Thanks Mike!